android/titanium/src/java/org/appcelerator/titanium/TiBlob.java

```diff
@@ -641,7 +641,7 @@ public TiBlob imageAsCropped(Object params)
 	@Kroll.method
 	public TiBlob imageAsResized(Number width, Number height)
 	{
-		boolean valid = (image != null) || (image == null && (this.width > 0 && this.height > 0));
+		boolean valid = (image != null) || (this.width > 0 && this.height > 0);
```

Comment:
Can you please go back with git-blame and see why the check was added in the first place? It looks like it was there for another reason

Reply:
it was added here: https://github.com/tidev/titanium_mobile/pull/5597/files#diff-a8746380cd1e01787f46def043161a8d66bb98fb0f1e5b8184bd981355f5d7c7R563

in a [TIMOB-1510 Memory Optimizations on TiBlob ](https://jira-archive.titaniumsdk.com/TIMOB-1510)
https://github.com/tidev/titanium_mobile/pull/5597/commits/efe48cbe9b10623561cb9b6c73d48e4e14e1632d

Reply:
btw: looks like I only used that for a different resize way before. It works fine keeping it in there